app/controllers/names/descriptions_controller.rb

```diff
@@ -92,11 +91,10 @@ def name_descriptions_by_author
 
     # Display list of name_descriptions that a given user is editor on.
     def name_descriptions_by_editor
-      user = if params[:by_editor]
-               find_or_goto_index(User, params[:by_editor].to_s)
-             else
-               @user
-             end
+      user = find_obj_or_goto_index(
+        model: User, obj_id: params[:by_editor].to_s,
+        index_path: name_descriptions_path
+      )
```

Comment:
Changes functionality when by_user is not an existing user.
See [Comment about LocationsController](https://github.com/MushroomObserver/mushroom-observer/pull/1324/files#r1095161355).